lua/terrortown/menus/gamemode/base_gamemodemenu/base_gamemodesubmenu.lua

```diff
@@ -0,0 +1,56 @@
+---
+-- @class CLGAMEMODESUBMENU
+
+CLGAMEMODESUBMENU.type = "base_gamemodesubmenu"
+CLGAMEMODESUBMENU.priority = 0
+CLGAMEMODESUBMENU.title = ""
+
+---
+-- This function is used to populate the submenu on open/refresh.
+-- @note This function should be overwritten but not not called.
+-- @param Panel parent The parent panel
+-- @internal
+-- @realm client
+function CLGAMEMODESUBMENU:Populate(parent)
+
+end
+
+---
+-- This function is used to populate the button panel of a submenu on open/refresh.
+-- @note This function should be overwritten but not not called.
```

Comment:
```suggestion
-- @note This function should be overwritten but not called.
```

This and on all other places